data-scraping/IntroDataScraping.Rmd

```diff
@@ -2,7 +2,13 @@
 title: "Introduction to Data scraping with R"
 output:
   html_document:
-    df_print: paged
+ html_document:
```

Comment:
@Tanvi-Jindal, I'm unable to knit this document and the error points to the YAML header. Please check this out so that we can knit. Thanks!

Reply:
I think if you delete the second `html_document:` this should fix it.